Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.7",
+	"GoVersion": "go1.9",
```

Comment:
I think we should keep this at `go1.7` for the 1.6 branch.

Reply:
done